Sources/Extensions/MapKit/MKPolylineExtensions.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  MKPolylineExtensions.swift
+//  SwifterSwift
+//
+//  Created by Shai Mishali on 3/8/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+import MapKit
```

Comment:
Looks like we'll have to wrap these extensions in `#if os(iOS) || os(macOS)`. We support every platform and it seems to be unsupported on `watchOS` and `tvOS < 9.2`

Reply:
I'm not entirely sure it builds it in watchOS given the fact its not part of that target 🤔 
![f296d0b7-85c9-4bb0-b7f6-8dbe4c9a37c4](https://user-images.githubusercontent.com/605076/37174797-9ec37aae-22e5-11e8-9d73-eca6120a687e.png)